tox.ini

```diff
@@ -22,6 +22,7 @@ pass_env =
     PYTEST_*
 set_env =
     COVERAGE_FILE = {work_dir}/.coverage.{env_name}
+    TOX_USER_CONFIG_FILE = /dev/null
```

Comment:
This makes no sense on Windows, what breaking change are you trying to stop?

Reply:
The presence of my own `/Users/ssbarnea/Library/Application Support/tox/config.ini` file does prevent successful run of this particular test. Still, based on its content other tests might be affected too. Basically this isolates tox-uv testing from local user tox config.

How you want to achieve it... up to you. The idea is that during testing, that file should not be looked at.

Reply:
I agree with the sentiment, just not the approach. 🤔 This should be some kind of pytest fixture or conftest.py on load change.

Reply:
Windows is supported, even though is not explcitiely tested in the CI. So we try our best to allow people with Windows laptop to contribute.

Reply:
> there are other problems that prevent this repo from running its test on windows anyway

Such as?